test/test_torch.py

```diff
@@ -16677,7 +16653,8 @@ def __init__(self,
                   ref_backend='scipy', rtol=0.0008, atol=1e-5),
                   _TorchMathTestMeta('digamma',
                   input_fn=_generate_gamma_input, inputargs=[True], ref_backend='scipy',
-                  replace_inf_with_nan=True)]
+                  replace_inf_with_nan=True),
+                  _TorchMathTestMeta('abs', input_fn=_medium_2d, dtypes=_types_no_half, rtol=0., atol=0.)]
```

Comment:
CUDA supports abs for half, right? Is that tested?

Reply:
The accuracy is tested in TestTensorDeviceOps, testing behavior wrt non-contiguous inputs for half specifically is not particularly useful. test_abs used to run on cpu only.